drivers/display/Kconfig.gc9x01x

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+# SPDX-License-Identifier: Apache-2.0
+
+config GC9X01X
+	bool "GC9X01X display driver"
+	default y
+	depends on DT_HAS_GALAXYCORE_GC9X01X_ENABLED
+	select SPI
+	select PWM
```

Comment:
drop?

Reply:
Fixed in [810703f](https://github.com/zephyrproject-rtos/zephyr/commit/810703f196aaa5242daf43eb983aeca14d0fb18b)